settings.py

```diff
@@ -923,6 +923,8 @@
 
 _location = lambda x: os.path.join(FILEPATH, x)
 
+IS_SASS_ENVIRONMENT = SERVER_ENVIRONMENT == 'production'
```

Comment:
ha, I thought this too...maybe a different name would be good. what does "sass" mean in this context?